### ISSUE TITLE
Using django-angular with django-bootstrap3

### ISSUE THREAD

**Issue by vatsalj** [jrief/django-angular#161]
Hi, 

I wanted to show a ModelForm with django-angular mixins in bootstrap style, for which I used [django-bootstrap3](https://github.com/dyve/django-bootstrap3), but it the two don't seem to be working together properly:

![image](https://cloud.githubusercontent.com/assets/1089034/7220813/fdb7b192-e6f2-11e4-90ee-d88a8ac0076e.png)

What can be done to make django-bootstrap3 work well with django-angular? Or is there any other django app that can be used instead of django-bootstrap3 that will work properly with django-angular?

**Comment by jrief** [comment]
I use both off them all the time, and I don't have this error.
Without reproducible setup, I don't know where to start digging.

**Comment by vatsalj** [comment]
@jrief I tried to find a solution by following the solution to Issue #51. Turns out, there's nowhere (except the source code) where the `Bootstrap3FormMixin` is mentioned (Not even in the docs) which is why I couldn't find it before raising this issue (Maybe it should be put docs as well?). That mixin takes care of everything! After seeing my angular forms rendering beautifully, I have to say that you've done an amazing job with django-angular. Thank you very much!
